x-pack/plugins/security_solution/public/common/store/data_table/discover.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Storage } from '@kbn/kibana-utils-plugin/public';
+import type { Action } from 'redux';
+import { updateDiscoverAppState } from '../discover/actions';
+import type { SecuritySolutionDiscoverState } from '../discover/model';
+import { initialDiscoverAppState } from '../discover/model';
+
+const discoverActionTypes = [updateDiscoverAppState.type];
+
+const LOCAL_STORAGE_DISCOVER_KEY = 'securitySolution.discover';
```

Comment:
Was it intentional to not place this as `securitySolution.timeline.discover`? In serverless, we also have a top level discover page so I would ideally want to be as specific as possible.

Reply:
your comment makes sense.. I will do the necessary changes.